spec/GraphQLOverHTTP.md

```diff
@@ -455,6 +472,9 @@ depending on the concrete error condition.
 
 Note: Typically this will be the `400` (Bad Request) status code.
 
+Note: This rule is "should" to maintain compatibility with legacy servers which
+may return 200 status codes even when this type of error occurs.
```

Comment:
This note implies a changing normative rule that doesn't exist.  Consider rewording.